### DISCUSSION TITLE
Welcome to EEP153_team_Malthus Discussions!

### DISCUSSION THREAD

**Discussion by chrisgnicholas** [Chris-Nicholas-collaborators/EEP153_team_Malthus#1] [Announcements]
## 👋 Welcome!
  Trying out Discussions as a place to connect with other members of our team. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chrisgnicholas** [comment]
Trying out github discussions as a way to talk about our project.

**Reply by t-pappas** [reply]
Thank you Chris! I am beginning to familiarize myself with github, and I will turn on notifications so I can see when people message on our github.

**Comment by samlwang2** [comment]
Before Friday please look at pre and post-1990 (year of Namibia's independence from South Africa) datasets. Look into aspects you may be interested in and create your branch. Play around with data and visuals to share.

**Comment by samlwang2** [comment]
1. [A] Population Statistics - Chris
2. [A] Unit tests - Tia
3. [A] Population DataFrames - Sam
4. [B] Population Pyramids - Angela
5. [C] Animated Population Pyramids - Steven
6. [C] Population Maps - Chris 
7. [C] Other Visualization Tools - Bolor
8. [C] Agricultural Supply - Bolor
9. [A] Presentation - Sam/Tia -> Bolor will check/edit
10. [A] GitHub Repository Containing Code + Code Review - Angela/Steven

**Reply by samlwang2** [reply]
Here are how we will divide and conquer -> please focus on tasks on 1-8 for now!!!
